octodns/processor/filter.py

```diff
@@ -234,8 +234,12 @@ def _process(self, zone, *args, **kwargs):
             if hasattr(record, 'values'):
                 values = [value.rdata_text for value in record.values]
             else:
-                values = [record.value.rdata_text]
-
+                try:
+                    values = [record.value.rdata_text]
+                except AttributeError:
+                    self.log.warning(
+                        f"Value processor ignoring record, value is NoneType: {record.fqdn}"
+                    )
```

Comment:
Since this is already doing `if hasattr(record, 'values'):` up above I think it'd make sense to use an elif to check for a non-None value too rather than add a 2nd level of branching with the try/except:

```python
elif  record.value is not None:
    values = [record.value.rdata_text]
else:
    self.log.warning('value for %s is NoneType, ignoring', record.fqdn)
```

Also reworked the log line. No need to include "Value processor" since the class name and ID will be the logger name. Also using placeholders rather than f-string as is the standard practice for log lines (in octoDNS.)